operator/src/network/ceramic.rs

```diff
@@ -141,17 +145,6 @@ pub struct CeramicSpec {
     pub ipfs: Option<IpfsSpec>,
     /// Resource limits for ceramic nodes, applies to both requests and limits.
     pub resource_limits: Option<ResourceLimitsSpec>,
-    /// Name of secret containing the private key used for signing anchor requests and generating
-    /// the Admin DID.
-    pub private_key_secret: Option<String>,
```

Comment:
There's a bit of a problem with this (independent of this PR and mostly my fault). Private keys/DIDs are not supposed to be shared across Ceramic nodes - each is supposed to use its own to prevent anchoring conflicts in case both nodes write to the same stream.

Reply:
We'd need to generate fresh keys for each node when the secret is unspecified and find a way to automatically authenticate with CAS Auth. We could also allow specifying a key per node but that could get messy and we'd need to check that keys don't collide, etc.

Feels like cleanest, easiest would be to generate and auth new keys. That would bloat the API key space but I don't think that's too much of an issue.

Reply:
ok, want to create an issue for that? Let fix it in a separate PR

Reply:
Yep: https://linear.app/3boxlabs/issue/RES-1206/create-and-authenticate-dids-for-keramik-ceramic-nodes